cluster-autoscaler/core/scaleup/orchestrator/orchestrator_test.go

```diff
@@ -1664,6 +1665,110 @@ func TestScaleUpToMeetNodeGroupMinSize(t *testing.T) {
 	assert.Equal(t, "ng1", scaleUpStatus.ScaleUpInfos[0].Group.Id())
 }
 
+func TestScaleupAsyncNodeGroupsEnabledWithNoUpcomingNodegroup(t *testing.T) {
```

Comment:
Can you make the new tests table-based? Ideally, combine with some of the existing ones if possible.